app/experimenter/templates/experiments/detail_base.html

```diff
@@ -17,6 +17,9 @@ <h3>
   </h3>
 
   {{ experiment.dates }}
+  {% if experiment.survey_required %}
+    <span class="badge badge-secondary">Includes Survey</span>
+  {% endif %}
 {% endblock %}
 
```

Comment:
Is the screenshot from experimenter channel what this ended up looking like? I am wondering if this pill could use the slightest bit of margin on the left..

Reply:
ooh you're right i should've put screen shots....